src/components/Authentication/Login.jsx

```diff
@@ -25,73 +27,87 @@ const Login = ({ cookies }) => {
   const [password, setPassword] = useState();
   const [showPassword, setShowPassword] = useState(false);
   const [errorMessage, setErrorMessage] = useState();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'errors' is assigned a value but never used.